### PR TITLE
Fix sonar issues

### PR DESCRIPTION
**Pull request by matt-k1998**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EM-4185

### Description ###
Fix SonarQube issues

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java [hunk 1]
```diff
@@ -89,7 +86,6 @@ private void uploadNewDocument(File file, CreateTemplateRenditionDto createTempl
             }
 
         } catch (RuntimeException | IOException e) {
-            log.error(e.getMessage(), e);
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java:92]: may be better to keep with `log.error("{description }", e)';

File: src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java [hunk 2]
```diff
@@ -101,8 +97,8 @@ private void uploadNewDocumentVersion(File file, CreateTemplateRenditionDto crea
                     .Builder()
                     .setType(MultipartBody.FORM)
                     .addFormDataPart("file", createTemplateRenditionDto.getFullOutputFilename(),
-                            RequestBody.create(MediaType.get(createTemplateRenditionDto.getOutputType().getMediaType()),
-                                    file))
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java:105]: this is a bug?

File: src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java [hunk 3]
```diff
@@ -120,7 +116,6 @@ private void uploadNewDocumentVersion(File file, CreateTemplateRenditionDto crea
             }
 
         } catch (RuntimeException | IOException e) {
-            log.error(e.getMessage(), e);
```
Comment (mdayican) [inline src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java:123]: same here `log.error("{description }", e)';

File: src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java (hunk 1 above)
Comment (matt-k1998) [reply src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java:92]: SonarQube complains about that (Exceptions should be either logged or rethrown but not both - java:S2139).

File: src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java (hunk 2 above)
Comment (matt-k1998) [reply src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java:105]: SonarQube was complaining that the create method is deprecated ("@Deprecated" code should not be used - 
java:S1874). I googled it and apparently swapping the order of the arguments fixes it - https://newbedev.com/how-to-replace-deprecated-okhttp-requestbody-create

File: src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java (hunk 1 above)
Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java:92]: strange we hame many like this logging and throwing another exception.

**Review by mdayican — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java (hunk 1 above)
Comment (matt-k1998) [reply src/main/java/uk/gov/hmcts/reform/dg/docassembly/service/DmStoreUploader.java:92]: maybe java:S2139 is suppressed in those places?

**Pull request merged**